core/extension/gdextension_interface.h

```diff
@@ -815,8 +815,11 @@ typedef void (*GDExtensionMainLoopShutdownCallback)();
 typedef void (*GDExtensionMainLoopFrameCallback)();
 
 typedef struct {
+	// Will be called after Godot is started and is fully initialized.
 	GDExtensionMainLoopStartupCallback startup_func;
+	// Will be called before Godot is shutdown when it is still fully initialized.
 	GDExtensionMainLoopShutdownCallback shutdown_func;
+	// Will be called for each process frame.
```

Comment:
> for each process frame.

Is there any temporal relation to `Node::_process` (always before, or always after)? If yes, would be good to document.